package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "formidable",
+  "name": "@coradine/formidable",
```

Comment:
I think we want our own package of this so I added `@coradine` so it doesn't try to release to the source repository we forked from.